src/components/AmountTextInput.js

```diff
@@ -17,7 +17,13 @@ class AmountTextInput extends React.Component {
   }
 
   focus = () => {
-    this.inputRef.current.focus();
+    /* After the focus method is called, the screen is still re-rendered at least once more.
```

Comment:
Why is it happening just now? It was working before, wasn't it?

Reply:
Yes, it was working before.
I couldn't find a decisive answer for why this stopped working, but it probably has something to do with the general changes to the [`focus` related functions from React Navigation 4 to 5](https://reactnavigation.org/docs/5.x/upgrading-from-4.x/#navigation-events).

Reply:
A few considerations about this:

- The `AmountTextInput` that needs to be focused here is mounted when the `My Address` tab is focused, and because of that is not available to be focused at mount time.
- The tab management component currently used does not use the React Navigation [`material-top-tab-navigator`](https://reactnavigation.org/docs/material-top-tab-navigator) wrapper, but instead uses directly its underlying [`react-native-tab-view`](https://reactnavigation.org/docs/tab-view/) lib.

That way, the `focus` event does not work as expected within the screen after the migration from Navigator 4 to 5 and 6. It only receives the `focus` event when changing from the _Send_ stack to the _Receive_ stack, for example.

I would suggest opening another PR to use the `material-top-tab-navigator` on this _Receive_ stack instead of the _tab view_, so that we could have the `focus` events available on the screens again through React _hooks_ and remove the need for this `setTimeout`.

Reply:
Ok. Create another issue to handle this and add it to the upgrade dependencies project, we can do it later